estate/models/estate_property.py

```diff
@@ -0,0 +1,40 @@
+from odoo import fields, models
+
+
+class Estate(models.Model):
+    _name = "estate.property"
+    _description = "Properties of estate entities."
+
+    name = fields.Char(required=True)
+    description = fields.Text()
+    active = fields.Boolean(default=True)
+    postcode = fields.Char()
+    date_availability = fields.Date(
+            copy=False,
+            default=fields.Date.add(fields.Date.today(), fields.Date.today(), months=3))
+    expected_price = fields.Float(required=True, readonly=True)
+    selling_price = fields.Float(copy=False, readonly=True)
+    bedrooms = fields.Integer(default=2)
+    living_area = fields.Integer()
+    facades = fields.Integer()
+    garage = fields.Boolean()
+    garden = fields.Boolean()
+    garden_area = fields.Integer()
+    garden_orientation = fields.Selection(
+            string='Orientation',
+            selection=[
+                ('North', 'north'),
+                ('South', 'south'),
+                ('East', 'east'),
+                ('West', 'east')],
```

Comment:
```suggestion
                ('north', 'North'),
                ('south', 'South'),
                ('east', 'East'),
                ('west', 'West')],
```
The list of tuple we give to selection is something like `('unique_technical_identifier', 'Human Readable Name')`